README.md

```diff
@@ -2,18 +2,18 @@
 Collaborative Classroom Norms for CS2 2019
 
 ## Norms
-I **learn** best when: 
+I **learn** best when: There are very few distractions
 
-So I want a classroom that is:
+So I want a classroom that is: Free of interruptions
 
-In order to acheive this, I proposet the following classroom norm:
+In order to acheive this, I propose the following classroom norm: Take care of your personal needs before coming to class on time.
```

Comment:
@fcsmakers I think that this is actually two different norms, one about taking care of your personal needs and one about arriving on time.